### DISCUSSION TITLE
AzCopy 7.3 question

### DISCUSSION THREAD

**Discussion by paxer2k** [community/community#126953] [Programming Help] [answered]
### Body

Greetings,

I have a specific question regarding AzCopy, particularly concerning the 7.3 version. I am using AzCopy to copy Azure Table Storage table information to Azure Blob Storage through JSON-based files. All of this is not an issue and I got it to work.

My question is related to how this copy happens. When AzCopy is in the process of copying the data over, does it make a snapshot or save the current state of the table? In other words if changes are being made to the Azure Table Storage table that is current in the process of being copied by AzCopy, will this change be included in that copy or not? 

I hope this question is clear, and if not, please let me know. Thank you for the response in advance!

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by paxer2k** [comment, marked as answer]
Update:

I might have found a potential answer to this: 

![image](https://github.com/community/community/assets/70654308/d28164cb-b0ae-4781-bd6b-f97d99cef7e9)


Source: https://learn.microsoft.com/en-us/previous-versions/azure/storage/storage-use-azcopy-linux

**Reply by ebndev** [reply]
@paxer2k thanks for being a part of the GitHub Community and posting your answer once you resolved it! 😄
